tsconfig.build.json

```diff
@@ -25,10 +25,8 @@
     "incremental": true,
     "preserveWatchOutput": true,
 
-
-    // There are two duplicate type definitions included from `chai` and `vitest` packages.
-    // There is one invalid type declaration introduced from `webdriverio -> got` package. 
-    // TODO: Once we completely remove `chai` and upgrade `webdriverio` we can enable this check again.  
+    // TODO: Investigate following errors:
+    // - Cannot find module 'rollup/parseAst' or its corresponding type declarations
```

Comment:
🤔

Reply:
We had this ts config earlier for the packages and reason listed in comments. I tried to remove and verify, so with new update packages the old error is no more there but a new one pop up. 

Investigating that would be a different separate task.

Reply:
pls create issues for this